### ISSUE TITLE
Google Assistant not working

### ISSUE THREAD

**Issue by NovaViper** [maykar/plex_assistant#125]
Hi, I just followed the guide that's on the new README.md for setting up Google Assistant with DialogFlow, however I can't get it working at all. I run Test Apps but I get this error. I have no idea what is going on or what I am doing wrong. I'm using Home Assistant Cloud for access to the internet.
![image](https://user-images.githubusercontent.com/7191115/118746004-01e0bc80-b81d-11eb-97dd-40aa2cf1ca74.png)

**Comment by Kisty** [comment]
I am having the same issue, however from Dialogflow test console it works fine.

**Comment by spicerackk** [comment]
Having similar issue. I am able to get it to (kind of) work from DialogFlow test console (it played a completely different movie) but I can't ask Google Home devices to play it, it only shows me imdb or google search results

**Comment by D34DC3N73R** [comment]
I think these steps need to be added to the dialogflow setup, 
- Click on integrations
- Under Google Assistant click "Not ready yet? Continue with the integration" 
- Click "Manage Assistant App"
- This opens up a new tab at console.actions.google.com for the Dialogflow project
- Click "Quick Setup" and "Decide how your action is invoked"
- Change the Display Name to "Plex"

**Comment by spicerackk** [comment]
I can get through as far as the fourth step, but when I open the new tab, I get an error simply saying "There was an error completing your request", with a button saying Try Again.

**Comment by D34DC3N73R** [comment]
Can you sign in to https://console.actions.google.com/ ?

**Comment by spicerackk** [comment]
yeah I can, takes me to the "Welcome to Actions on Google" page.

**Comment by D34DC3N73R** [comment]
No idea then. I had some previous projects in there, but the plex project was created automatically for me when clicking the link in Dialogflow.

**Comment by spicerackk** [comment]
weird, I'll try refresh everything and try again

**Comment by spicerackk** [comment]
Still no luck. Should I build a new project in Action Blocks?

**Comment by spicerackk** [comment]
ok so it seemed I had to import the plex project before it worked, I'm yet to test it though.

**Comment by D34DC3N73R** [comment]
Yes the steps I outlined would come after the initial Dialogflow steps outlined in the documentation.

**Comment by spicerackk** [comment]
So if I ask GA to "Play The Avengers on Plex on the Office Display", it responds with "I don't know, but I found these results on search" and pulls up search results. Still doesn't seem to work after doing the above steps.

**Comment by D34DC3N73R** [comment]
You need to use the specific phrases outlined in the documentation. "Hey Google, tell plex to play the avengers on the office display"

"Tell Plex" or "Ask Plex" should immediately follow "Ok Google" or "Hey Google".
